tests/AppBundle/Model/EventWikiTest.php

```diff
@@ -147,5 +147,9 @@ public function testPages(): void
         $wiki->setPagesEdited([4, 5, 6]);
         static::assertEquals([4, 5, 6], $wiki->getPagesEdited());
         static::assertEquals([1, 2, 3, 4, 5, 6], $wiki->getPages());
+
+        // Make sure that empty strings are removed.
+        $wiki->setPagesCreated(['']);
```

Comment:
I think this test would be more representative if you added empties in between numbers, as it will usually come from the database?

Also, are we really expecting numbers, or strings?

Reply:
Yeah, good idea. I think we never do save anything other than arrays of integers, but "be liberal in what you accept" and all that...

I'll update it.